src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -78,10 +79,20 @@ std::string postEntities
     ciP->httpHeaderValue.push_back(location);
     ciP->httpStatusCode = SccCreated;
   }
+  else
+  {
+    OrionError oe;
+    oe.code = ciP->httpStatusCode;
+    oe.reasonPhrase = "AlreadyExists";
+    oe.details = "An entity with the same id and type "
+                 "already exists. You can use POST /entities/{entityId} "
```

Comment:
Although this is the text that appear #1158, thinking a little bit on it, it could be missleanding as:

* The operation causing the problem could not include a type (i.e. only including id)"
* It is not fully cleear if the suggested "POST /entities/{entityId}" could include URI params and, in that case, if the user includes `op=append", the problem happens again.

Thus, maybe is better to use jsut:

```
oe.details = "Entity already exists"
```

(I will update #1158)